packages/components/src/components/TokenInput/tokens.ts

```diff
@@ -0,0 +1,24 @@
+import { Token } from './TokenInput';
+
+export const defaultTokens: Token[] = [
+  {
+    symbol: 'USDT',
+    address: '0x55d398326f99059fF775485246999027B3197955',
+  },
+  {
+    symbol: 'DAI',
+    address: '0x6B175474E89094C44Da98b954EedeAC495271d0F',
+  },
+  {
+    symbol: 'ETH',
+    address: '0xc3761eb917cd790b30dad99f6cc5b4ff93c4f9ea',
+  },
+  {
+    symbol: 'GTC',
+    address: '0xde30da39c46104798bb5aa3fe8b9e0e1f348163f',
+  },
+  {
+    symbol: 'USDC',
+    address: '0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48',
+  },
+];
```

Comment:
Let's not include any default tokens at all. Make `tokens` a required prop please.

Reply:
Yes, I've updated the component to accept the `tokens` as a required prop. I've kept this to use in the stories.
Also, I've updated the `tokens` prop to `tokenList` prop.